dataprofiler/profilers/profile_builder.py

```diff
@@ -2112,6 +2130,174 @@ def _update_chi2(self):
 
         return chi2_mat
 
+    def _update_null_replication_metrics(self, clean_samples):
+        """
+        Calculate metrics needed for replicating null values in capitalone/synthetic-data.
+
+        Required for running LDA based binary classifier where predicted class label indicates whether
+        a value of a column should be NaN (1) or not (0).
+
+        :param clean_samples: input cleaned dataset
+        :type clean_samples: dict
+        """
+        data = pd.DataFrame(clean_samples).apply(pd.to_numeric, errors="coerce")
```

Comment:
is this call still needed? I assume yes for the conversion to numeric? Do we need to apply the entire array? is this expensive?

Reply:
Honestly, not too sure about that. Doing it because that is what correlation calculation does and I need the same type of data for null replication as correlation calculation uses.